lib/Subscribers/ImportExport/Import/MailChimp.php

```diff
@@ -124,6 +126,7 @@ function getAPIKey($api_key) {
   }
 
   function throwException($error) {
+    $errorMessage = __('Unknown MailChimp error message.', 'mailpoet');
```

Comment:
Just `Unknown MailChimp error` sounds a bit better to me.

Reply:
Souds good, thanks, will update it.